sdk.js

```diff
@@ -324,6 +324,35 @@ class PaddleSDK {
 		});
 	}
 
+	/**
+	 * Update subscription details, quantity, price and or currency
+	 *
+	 * @method
+	 * @param {number} subscriptionID
+	 * @param {Object} postData { quantity, price, planID, currency }
+	 * @returns {Promise}
+	 * @fulfill {object} - The result of the operation
+	 *
+	 * @example
+	 * const result = await client.updateSubscriptionPlan(123, { quantity: 2 });
+	 */
+	updateSubscription(subscriptionID, postData) {
+		const { quantity, price, planID, currency } = postData;
+		const body = {
+			subscription_id: subscriptionID,
+			quantity,
+			price,
+			plan_id: planID,
+			currency,
+		};
+		// remove all empty params
+		Object.keys(body).forEach(k => body[k] == undefined && delete body[k]);
```

Comment:
Instead of doing this, let's build the body object with conditionals and only include valid values.

Reply:
done.  noticed a bug too!